test/e2e_node/pod_conditions_test.go

```diff
@@ -206,6 +242,10 @@ func getTransitionTimeForPodConditionWithStatus(pod *v1.Pod, condType v1.PodCond
 
 func webserverPodSpec(podName, containerName, initContainerName string, addInitContainer bool) *v1.Pod {
 	p := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
```

Comment:
this is not needed

Reply:
good catch. will update soon. I was setting this field while investigating how to use ownerReference and localObjectReferece to combine the configmap we missing here.